app/assets/stylesheets/clean-blog.css

```diff
@@ -0,0 +1,410 @@
+/*!
```

Comment:
Do you really need all of these css rules?

Reply:
I removed some css rules, but I included the majority to give best style to buttons, text area, inputs, header, and navbar. And also these rules handle the style for small screens.